doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md

```diff
@@ -0,0 +1,193 @@
+# Batch Plotting and Using EMCPy Backend for eva
+
+In addition to the `datasets` and `transforms` section within the configuration YAMLs used within eva, there is a `graphics` section. This section is used to design the figures needed during analysis of your data using EMCPy's plotting backend. 
+
+An example of a `graphics` section from a test configuration file for both a scatter plot and map scatter plot is shown below:
+
+``` yaml
+graphics:
+
+  # Scatter plot
+  # ------------
+
+  # JEDI h(x) vs Observations
+  - batch figure:
+      variables: *variables
+    figure:
+      layout: [1,1]
+      title: 'Observations vs. JEDI h(x) | Aircraft | ${variable_title}'
+      output name: observation_scatter_plots/aircraft/${variable}/jedi_hofx_vs_obs_aircraft_${variable}.png
+    plots:
+      - add_xlabel: 'Observation Value'
+        add_ylabel: 'JEDI h(x)'
+        add_grid:
+        add_legend:
+          loc: 'upper left'
+        layers:
+        - type: Scatter
+          x:
+            variable: experiment::ObsValue::${variable}
+          y:
+            variable: experiment::hofx::${variable}
+          markersize: 5
+          color: 'black'
+          label: 'JEDI h(x) versus obs (all obs)'
+        - type: Scatter
+          x:
+            variable: experiment::ObsValue::${variable}
+          y:
+            variable: experiment::hofxPassedQc::${variable}
+          markersize: 5
+          color: 'red'
+          label: 'JEDI h(x) versus obs (passed QC in JEDI)'
+
+  # Map plot
+  # ---------
+
+  # Observations
+  - batch figure:
+      variables: *variables
+    dynamic options:
+      - type: vminvmaxcmap
+        data variable: experiment::ObsValue::${variable}
+    figure:
+      figure size: [20,10]
+      layout: [1,1]
+      title: 'Observations | Aircraft | Obs Value'
+      output name: map_plots/aircraft/${variable}/observations_aircraft_${variable}.png
+    plots:
+      - mapping:
+          projection: plcarr
+          domain: global
+        add_map_features: ['coastline']
+        add_colorbar:
+          label: ObsValue
+        add_grid:
+        layers:
+        - type: MapScatter
+          longitude:
+            variable: experiment::MetaData::longitude
+          latitude:
+            variable: experiment::MetaData::latitude
+          data:
+            variable: experiment::ObsValue::${variable}
+          markersize: 2
+          label: ObsValue
+          colorbar: true
+          cmap: ${dynamic_cmap}
+          vmin: ${dynamic_vmin}
+          vmax: ${dynamic_vmax}
+            
+```
+
+## Batch Figure
+
+Given the variables selected in the `datasets` section of the configuration yaml, the user can create batch figures for all variables selected. In this example, a scatter plot and map plot will be created for all the variables the user selected to be analyzed.
```

Comment:
```suggestion
Given the variables selected in the `datasets` section of the configuration YAML, the user can create batch figures for all variables selected. In this example, a scatter plot and map plot will be created for all the variables the user selected to be analyzed.
```
I think we've been capitalizing YAML most places including elsewhere on this page.